components/Spaces/CallTabs/Music/SpotifySignUpBlocker.js

```diff
@@ -0,0 +1,51 @@
+import { useRouter } from 'next/router';
+import PropTypes from 'prop-types';
+import Backdrop from '@material-ui/core/Backdrop';
+import Button from '@material-ui/core/Button';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import { makeStyles } from '@material-ui/core/styles';
+import { useRecoilValue } from 'recoil';
+
+import * as spotifyState from '@/atoms/spotify';
+
+const useStyles = makeStyles((theme) => ({
+  backdrop: {
+    width: '100%',
+    height: '100%',
+    position: 'relative',
+    zIndex: theme.zIndex.drawer + 1,
+    color: '#fff',
+  },
+}));
+
+export const ROOM_ID = 'ROOM_ID';
+
+const SpotifySignUpBlocker = ({ loading }) => {
+  const classes = useStyles();
+  const roomID = useRecoilValue(spotifyState.roomID);
+  const spotifyAuthURL = useRecoilValue(spotifyState.spotifyAuthURL);
+  const router = useRouter();
+
+  const handleClicked = () => {
+    document.cookie = `productify_roomID=${roomID}; path=/`;
+    router.push(spotifyAuthURL);
+  };
+
+  return (
+    <Backdrop className={classes.backdrop} open>
+      {loading ? (
+        <CircularProgress color="secondary" />
+      ) : (
+        <Button variant="outlined" color="secondary" onClick={handleClicked}>
+          Sign in to Spotify
+        </Button>
+      )}
+    </Backdrop>
+  );
+};
+
+SpotifySignUpBlocker.propTypes = {
+  loading: PropTypes.bool.isRequired,
+};
+
```

Comment:
ye boiii it works
![image](https://user-images.githubusercontent.com/39652186/130380480-b920ee47-5686-44a2-9024-aad5d89cfd12.png)
![image](https://user-images.githubusercontent.com/39652186/130380519-7320f625-31d0-4eb7-95f0-a9251dddf3eb.png)